src/common/utils/auth.ts

```diff
@@ -0,0 +1,7 @@
+import authService from '../../features/auth/authService';
+
+export const getProfileToken = () => {
```

Comment:
Optional: could this be simply moved to the `AuthService` itself?

Reply:
Yup, I will move it in a follow-up PR.